modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -63,8 +65,9 @@ def get_pdf_data
           }
         end
 
-        def evss_service
-          ClaimsApi::EVSSService::Base.new(request)
+        # Creates a token OR gets existing one
+        def get_auth_token_service
+          @auth_token ||= BenefitsDocuments::Service.new.get_auth_token
```

Comment:
This service ultimately calls a service that looks up a token in redis, and if it finds one, returns it, or it will make a call to the auth server to get a new token. 

If we look at the other `lib/lighthouse/benefits_documents/configuration.rb`, we see that the last value provided in line :129 to the `Auth::ClientCredentials::Service` is the same as what's provided in our configuration file below. Perusing that `Auth::ClientCredentials::Service`, we can see that during initialization, that gets set to `@service_name` which is used to locate a token in redis. This will cause some issues with authorizing the "correct" user as our team and VA Gov will essentially "share" a token.

Reply:
Yep that makes sense, this was what I was asking about before with the token. I thought we would want to do that but this explanation makes it clear why that is wrong.   Thanks for that I will update this so the token is specific for the `claims_api`.